src/consensus/validation.h

```diff
@@ -76,6 +80,17 @@ class CValidationState {
     }
     virtual unsigned char GetRejectCode() const { return chRejectCode; }
     virtual std::string GetRejectReason() const { return strRejectReason; }
+
+    virtual bool IsInvalidTx() const {
+        return txInvalidTx.IsNull() == false;
+    }
+    virtual CTransaction GetInvalidTx() const {
```

Comment:
Is there any reason why these two functions are grouped together or should they be separate?
```suggestion
    }
    
    virtual CTransaction GetInvalidTx() const {
```

Reply:
@johnhanlon86 what do you mean?

Reply:
@blondfrogs Just checking if you want whitespace/line-breaks between functions... more of a styling question than a technical one :)

Reply:
Oh doesn't matter atm